app/lib/common/widgets/share/widgets/external_share_options.dart

```diff
@@ -23,9 +25,16 @@ class ExternalShareOptions extends StatelessWidget {
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     final lang = L10n.of(context);
 
+    final isWhatsAppInstalled =
+        ref.watch(isAppInstalledProvider(ExternalApps.whatsApp)).valueOrNull;
+    final isTelegramInstalled =
+        ref.watch(isAppInstalledProvider(ExternalApps.telegram)).valueOrNull;
+    final isSignalInstalled =
+        ref.watch(isAppInstalledProvider(ExternalApps.signal)).valueOrNull;
+
```

Comment:
I'd move that `== true` check up here... just makes more sense to me ...

Reply:
Sure, let me change it accordingly.